### ISSUE TITLE
How to use it when testing

### ISSUE THREAD

**Issue by zhulanyun** [vacancy/Synchronized-BatchNorm-PyTorch#33]
I'm not sure whether I need to use the function convert_to when testing. Besides, can model.eval() switches it to the test situation?

**Comment by vacancy** [comment]
Yes, `model.eval()` will switch the BN mode.
